src/lib/component/partial/HandlerDetailsContainer/HandlerDetailsConfiguration.js

```diff
@@ -0,0 +1,213 @@
+import React from "/vendor/react";
+import PropTypes from "prop-types";
+import gql from "/vendor/graphql-tag";
+
+import {
+  Card,
+  CardContent,
+  Divider,
+  Grid,
+  Typography,
+} from "/vendor/@material-ui/core";
+
+import { Maybe, NamespaceLink } from "/lib/component/util";
+
+import {
+  Code,
+  CodeBlock,
+  CodeHighlight,
+  Duration,
+  Dictionary,
+  DictionaryKey,
+  DictionaryValue,
+  DictionaryEntry,
+} from "/lib/component/base";
+
+import LabelsAnnotationsCell from "/lib/component/partial/LabelsAnnotationsCell";
+
+const HandlerDetailsConfiguration = ({ handler }) => (
+  <Card>
```

Comment:
I wonder if we should make it a convention to always define a full function body for functional components. It would make for a much smaller diff in the event that we need to add hooks or other logic above the returned elements.

Reply:
I think that's a good idea. I find myself adding a function body often in development anyway.